compiler/src/iree/compiler/Codegen/Dialect/IREECodegenAttrs.td

```diff
@@ -38,20 +38,22 @@ def LLVMGPU_WarpReduction : I32EnumAttrCase<"LLVMGPUWarpReduction", 14>;
 def LLVMGPU_PackUnPack : I32EnumAttrCase<"LLVMGPUPackUnPack", 15>;
 def LLVMGPU_MatmulTensorCoreMmaSync : I32EnumAttrCase<"LLVMGPUMatmulTensorCoreMmaSync", 16>;
 
+def SPIRV_BaseLowering
```

Comment:
`SPIRVDefault` to be consistent with the CPU and GPU (and VMVX) defaults?

Reply:
Default is not descriptive to me--it says nothing about what the pipeline does. Also here this pipeline is not really the default; it's actually a special case. The default one should be BaseVectorize, given that's the most commonly picked one. So I prefer to stay this name to be precise.